src/function/chatPostMessage.js

```diff
@@ -0,0 +1,72 @@
+import https from "https";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+// Request header
+const HEADERS = {
```

Comment:
This is very obvious and I don't see any reason to have a comment here.
I agree to have some comments on your code. but not every lines or blocks. Please reconsider all comments and codes weather the comments are necessary there.